### PR TITLE
Create a Page for the user selected Game

### LINKED ISSUE #6: Create components
Create components for the App

- Button component
- Title component
- Dropdown component
- Filter component

### LINKED ISSUE #15: Create Game Page
As a user, I would like to have a page where I can see games

Possible solution:

image

To do:
- Fetch necessary data
- Make a component for game

### PR DESCRIPTION
**Pull request by markobrkovic**
Create missing main components needed for this page:
  - Create Title Component
  - Create Image Component
  - Create Line Component for design purposes
fixes #6 

Modify existing components:
  - Add a prop variable to some Components, for the purpose of adding classes to them from the Page Component

Create Page:
  - Make basic Page Layout with created Components
  - Edit the Backend API Fetch to send some basic data needed for the Page
  - Insert retrieved data to Components or basic HTML Elements on the Page
  - Style the Page
fixes #15     

Modify Storybook:
  - Remove Storybook's body padding

Install http-proxy-middleware:
  - npm install --save http-proxy-middleware
  - Use the middleware for Storybook
    
Edit Frontend fetch API:
  - Remove a part of URL

Current state of the page:
image

### CODE REVIEW

File: src/Pages/Game/Game.module.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.gameInfoContainer {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 3;
+}
+
+.gameTitle {
+  font-size: var(--large);
```
Comment (AnkePfei2fer) [inline src/Pages/Game/Game.module.css:29]: For better understanding, you could change the variable naming to --font-size-large ...

**Review by AnkePfei2fer — APPROVED** [review]
LGTM 🎨 Very impressive implementation of your design!
Just a small comment

File: src/Pages/Game/Game.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+export default function Game() {
+  const [game, setGame] = useState<null | GameProps>(null);
+
+  useEffect(() => {
+    async function getName() {
+      const gameData = await fetchGameInfo();
+      setGame(gameData);
+    }
+
```
Comment (davidwobido) [inline src/Pages/Game/Game.tsx:33]:
```suggestion
```

File: src/Pages/Game/Game.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+          <span className={styles.releaseDate}>{game?.release_dates[0].y}</span>
+        </div>
+      </section>
+      <Title
+        className={styles.gameSectionsTitle}
+        title="About this game"
+        size="h3"
+        weight="thin"
+      />
+      <Line width="half" highestOpacityPoint="start" />
```
Comment (davidwobido) [inline src/Pages/Game/Game.tsx:74]: Does this work?

File: src/components/Input/Input.module.css [hunk 4]
```diff
@@ -21,13 +21,13 @@
...
 }
 
 .medium {
-  font-size: var(--medium);
+  font-size: var(--font-size-medium);
 }
 
 .large {
-  font-size: var(--large);
+  font-size: var(--font-sizelarge);
```
Comment (davidwobido) [inline src/components/Input/Input.module.css:32]:
```suggestion
  font-size: var(--font-size-large);
```

**Review by davidwobido — APPROVED** [review]
lgtm! Great!

File: src/Pages/Game/Game.tsx (hunk 3 above)
Comment (markobrkovic) [reply src/Pages/Game/Game.tsx:74]: Yes :)

**Pull request merged**